Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Controllers/MembershipsController.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Know_Your_Nation_Speedy.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Know_Your_Nation_Speedy.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class MembershipsController : ControllerBase
+    {
+        private readonly MyDbContext _db;
+        readonly IConfiguration _config;
+        public MembershipsController(MyDbContext context, IConfiguration config)
+        {
+            _db = context;
+            _config = config;
+        }
+
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<Memberships>>> Get()
+        {
+            return await _db.MembershipsEntries.ToListAsync();
+        }
+
+        [HttpPost("CreateEditMembership")]
```

Comment:
This function does not update memberships, it only creates them, why call it Edit as well?

Reply:
This function will be edited modified completely, it is now to test front ends add functionality

Reply:
Its never good to name functions based on planned future implementations, it will happen sometimes that you forget about it and then the name stays like that forever. It makes it especially hard for people trying to read your code, because the function acts like it does something when in reality it doesn't.

Reply:
I do get this, however according to our wire frame this is one button, my task is to complete this whole integration, so will get to it as soon as i can confirm the create works